src/refcard/intro.md

```diff
@@ -14,69 +14,85 @@ around a binary and getting information about it.
+| afl                     | List functions                          |
+| afvl                    | List function local vars and args       |
+| avg                     | List globals                            |
+| t                       | List types                              |
+| iz[z]                   | List strings (two `z` for more)         |
+| pdf @ [funcname](Tab)   | Disassemble function (main, fcn, etc)   |
 | x [nbytes]              | Hexdump of nbytes, $b by default        |
-| aa                      | Auto analyze                            |
-| pdf@ [funcname](Tab)    | Disassemble function (main, fcn, etc.)  |
-| f fcn(Tab)              | List functions                          |
-| f str(Tab)              | List strings                            |
-| fr [flagname] [newname] | Rename flag                             |
-| psz [offset]~grep       | Print strings and grep for one          |
-| axF [flag]              | Find cross reference for a flag         |
+| wx [nbytes]             | Write hexadecimal string                |
+| axt @ [flag/address]    | Find cross reference to a flag/address  |
 
 ## Flags
 
-Flags are like bookmarks, but they carry some extra information like size, tags or associated flagspace. Use the `f` command to list, set, get them.
+Flags are like bookmarks, but they carry some extra information like size, tags or associated flagspace. Use `f` commands to list, set, get them.
 
 | Command             | Description           |
 |:--------------------|:----------------------|
-| f                   | List flags            |
+| f name              | Add flag "name"       |
+| f- name             | Remove flag "name"    |
+| fl                  | List flags            |
 | fd $$               | Describe an offset    |
-| fj                  | Display flags in JSON |
-| fl                  | Show flag length      |
+| fN [name]           | Show the real name    |
 | fx [flagname]       | Show hexdump of flag  |
 | fC [name] [comment] | Set flag comment      |
 
-## Flagspaces
+## Functions and variables
 
-Flags are created into a flagspace, by default none is selected,
-and listing flags will list them all. To display a subset of flags
-you can use the `fs` command to restrict it.
+Functions appear after auto-analysis or after adding them manually, e.g. with `af` command.
 
-| Command       | Description           |
-|:--------------|:----------------------|
-| fs            | Display flagspaces    |
-| fs *          | Select all flagspaces |
-| fs [space]    | Select one flagspace  |
+| Command             | Description               |
+|:--------------------|:--------------------------|
+| af                  | Analyze function          |
+| af- name            | Remove function "name"    |
+| afb                 | List basic blocks         |
+| afi                 | Show function information |
+| afs                 | Show function signature   |
+| afvl                | Show function variables   |
```

Comment:
maybe is worth mentioning agf for the function block graph

Reply:
Added